### DISCUSSION TITLE
Error during EBSD data plot in Mtex 5.4.0

### DISCUSSION THREAD

**Discussion by zaid442** [mtex-toolbox/mtex#549] [Ask Anything]
Hii 

I wanna jump to Mtex by following Mtex tutorials from https://mtex-toolbox.github.io/

  following is the error and I am clueless about it. Kindly do the needful 

 plot(EBSD)
>> EBSD
 
ans = EBSD (show methods, plot)
  EBSD is empty!
>> plot(EBSD('Iron bcc(old)')
 plot(EBSD('Iron bcc(old)')
                           ↑
Error: Invalid expression. When calling a function or indexing a variable, use parentheses. Otherwise, check for mismatched
delimiters.
 
Did you mean:
>> plot(EBSD('Iron bcc(old)'))
Error using rotation (line 107)
Wrong rotation syntax!

Error in EBSD (line 95)
      ebsd.rotations = rotation(rot);

Thanks in advance

**Comment by ralfHielscher** [comment]
Hi,

Matlab is case sensitive so you should use lower  case letters `ebsd`. Also make sure you have loaded some data.

Ralf

**Reply by zaid442** [reply]
HI Ralf, 

Thank you for the reply.

The error occurred after I load .ctf file using import_wizard(EBSD) command.  

if I use lower case import_wizard(ebsd) , it gives error and ask do u mean import_wizard(EBSD).

then I try to call Iron bcc(old) phase , 

I receive rotation error 

Kindly let me know why such error should occur even if I uploaded the data.

**Reply by ralfHielscher** [reply]
The command for calling the import wizard to import EBSD data is  
```matlab
import_wizard('EBSD')
```
once you have imported your data you can plot them with
```matlab
plot(ebsd)
```
make sure you read https://mtex-toolbox.github.io/EBSDTutorial.html 

Ralf
